out_writeapi.go

```diff
@@ -629,20 +664,27 @@ func FLBPluginExitCtx(ctx unsafe.Pointer) int {
 		log.Printf("Finding configuration for output instance with id: %d failed in FLBPluginExitCtx", id)
 		return output.FLB_ERROR
 	}
+	streamSlice := *config.managedStreamSlice
 
-	_, responseErr := checkResponses(ms_ctx, config.appendResults, true, &config.mutex, config.exactlyOnce)
-	if responseErr != nil {
-		log.Printf("Checking append responses for output instance with id: %d failed in FLBPluginExitCtx: %s", id, responseErr)
-		return output.FLB_ERROR
+	// TODO: Change for desired stream, defaulting to 0 for now
+	streamIndex := 0
```

Comment:
Why do we need this? Don't we have to iterate over all streams in the slice?

Reply:
Done.